### PR TITLE
Turn on more integration testing

### PR DESCRIPTION
**Pull request by jkozdon**

### CODE REVIEW

**Comment by jkozdon** [comment]
bors try

**Comment by jkozdon** [comment]
cc @lcw

**Comment by jkozdon** [comment]
bors try

**Comment by kpamnany** [comment]
This is great. Please let me know if any of these should be in the performance tests also, alongside Dycoms, for SlurmCI testing.

File: .slurmci/test_set.jl [hunk 1]
```diff
@@ -29,10 +26,12 @@ cpu_gpu_tests = Set(((3, "examples/DGmethods_old/ex_001_periodic_advection.jl"),
                      (2, "test/DGmethods_old/sphere/advection_sphere_lsrk.jl"),
                      (2, "test/DGmethods_old/sphere/advection_sphere_ssp33.jl"),
                      (2, "test/DGmethods_old/sphere/advection_sphere_ssp34.jl"),
-                     (2, "test/LinearSolvers/poisson.jl"),
                      (4, "examples/DGmethods/ex_001_dycoms.jl"),
                      (1, "test/Ocean/shallow_water/GyreDriver.jl")
```
Comment (simonbyrne) [inline .slurmci/test_set.jl]:
```suggestion
                     (1, "test/Ocean/shallow_water/GyreDriver.jl"),
```

File: test/DGmethods/integral_test_sphere.jl [hunk 2]
```diff
@@ -130,7 +129,7 @@ let
   expected_result = [6.228615762850257e-7
                      9.671308320438864e-9
                      1.5102832678375277e-10
-                     2.359860999112363e-12]
+                     2.3598612210570662e-12]
```
Comment (simonbyrne) [inline test/DGmethods/integral_test_sphere.jl]: Slurm doesn't like this change: 
https://gist.github.com/climabot/f8b1957d388a35030d5bcc272c22e0dd#file-out_7303963-L43

Comment (simonbyrne) [reply test/DGmethods/integral_test_sphere.jl]: @jkozdon any reason for this change?

Comment (simonbyrne) [reply test/DGmethods/integral_test_sphere.jl]:
```suggestion
                     2.359860999112363e-12]
```

**Comment by simonbyrne** [comment]
bors try

File: test/DGmethods/integral_test_sphere.jl (hunk 2 above)
Comment (simonbyrne) [reply test/DGmethods/integral_test_sphere.jl]: Ah, so the issue is that the CPU and GPU give different answers. Is this simply an order-of-operations issue, or something more serious. Can we just loosen the tolerance?

Comment (kpamnany) [reply test/DGmethods/integral_test_sphere.jl]: Parallel vs serial reduction perhaps? If so, typical solution is to loosen the tolerance.

**Pull request closed without merging**